src/main/scala/fs2/Chunk.scala

```diff
@@ -29,6 +29,16 @@ trait Chunk[+A] { self =>
     iterator.map(f).copyToBuffer(buf)
     Chunk.indexedSeq(buf)
   }
+  /**
+   * `[[scanLeft]]` is a helper for process1.scan; it differs
+   * from standard collection behavior by not including
+   * the initial value in the output.
```

Comment:
@pchiusano Behavior is consistent, as shown below:

* scan1

```
scala> Process.empty[Task, Int].scan1(_ + _).runLog.run
res7: IndexedSeq[Int] = Vector()

scala> Process(1).scan1(_ + _).toSource.runLog.run
res8: IndexedSeq[Int] = Vector(1)

scala> Process(1, 2).scan1(_ + _).toSource.runLog.run
res9: IndexedSeq[Int] = Vector(1, 3)

scala> Process(1, 2, 3).scan1(_ + _).toSource.runLog.run
res10: IndexedSeq[Int] = Vector(1, 3, 6)
```

```
scala> Stream.empty[Task, Int].scan1(_ + _).runLog.run.run
res2: Vector[Int] = Vector()

scala> (Stream(1): Stream[Task, Int]).scan1(_ + _).runLog.run.run
res3: Vector[Int] = Vector(1)

scala> (Stream(1, 2): Stream[Task, Int]).scan1(_ + _).runLog.run.run
res4: Vector[Int] = Vector(1, 3)

scala> (Stream(1, 2, 3): Stream[Task, Int]).scan1(_ + _).runLog.run.run
res5: Vector[Int] = Vector(1, 3, 6)
```

* scan

```
scala> Process.empty[Task, Int].scan(0)(_ + _).runLog.run
res11: IndexedSeq[Int] = Vector(0)

scala> Process(1).scan(0)(_ + _).toSource.runLog.run
res12: IndexedSeq[Int] = Vector(0, 1)

scala> Process(1, 2).scan(0)(_ + _).toSource.runLog.run
res13: IndexedSeq[Int] = Vector(0, 1, 3)

scala> Process(1, 2, 3).scan(0)(_ + _).toSource.runLog.run
res14: IndexedSeq[Int] = Vector(0, 1, 3, 6)
```

```
scala> Stream.empty[Task, Int].scan(0)(_ + _).runLog.run.run
res6: Vector[Int] = Vector(0)

scala> (Stream(1): Stream[Task, Int]).scan(0)(_ + _).runLog.run.run
res7: Vector[Int] = Vector(0, 1)

scala> (Stream(1, 2): Stream[Task, Int]).scan(0)(_ + _).runLog.run.run
res8: Vector[Int] = Vector(0, 1, 3)

scala> (Stream(1, 2, 3): Stream[Task, Int]).scan(0)(_ + _).runLog.run.run
res9: Vector[Int] = Vector(0, 1, 3, 6)
```

Most importantly with fold/fold1 is what happens with the empty case:

* fold1

```
scala> Process.empty[Task, Int].fold1(_ + _).runLog.run
res15: IndexedSeq[Int] = Vector()

scala> Process(1).fold1(_ + _).toSource.runLog.run
res16: IndexedSeq[Int] = Vector(1)

scala> Process(1, 2).fold1(_ + _).toSource.runLog.run
res17: IndexedSeq[Int] = Vector(3)
```

```
scala> Stream.empty[Task, Int].fold1(_ + _).runLog.run.run
res10: Vector[Int] = Vector()

scala> (Stream(1): Stream[Task, Int]).fold1(_ + _).runLog.run.run
res11: Vector[Int] = Vector(1)

scala> (Stream(1, 2): Stream[Task, Int]).fold1(_ + _).runLog.run.run
res12: Vector[Int] = Vector(3)
```

* fold

```
scala> Process.empty[Task, Int].fold(0)(_ + _).toSource.run
res19: IndexedSeq[Int] = Vector(0)

scala> Process(1).fold(0)(_ + _).toSource.runLog.run
res20: IndexedSeq[Int] = Vector(1)

scala> Process(1, 2).fold(0)(_ + _).toSource.runLog.run
res21: IndexedSeq[Int] = Vector(3)
```

```
scala> Stream.empty[Task, Int].fold(0)(_ + _).runLog.run.run
res13: Vector[Int] = Vector(0)

scala> (Stream(1): Stream[Task, Int]).fold(0)(_ + _).runLog.run.run
res14: Vector[Int] = Vector(1)

scala> (Stream(1, 2): Stream[Task, Int]).fold(0)(_ + _).runLog.run.run
res15: Vector[Int] = Vector(3)
```